### PR TITLE
docs: add documentation for metamanager

### LINKED ISSUE #100: Add documentation for edge core modules
This is an umbrella issue to describe how each beehive module in kubeedge works.
The expected output is a document which details the responsibilities of the module , its interfaces with other modules etc.

### PR DESCRIPTION
**Pull request by rohitsardesai83**
this adds the operations performed by metamanager

Partially fixes #100

### CODE REVIEW

File: docs/modules/metamanager.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+The insert operation request is received via the cloud by edgehub. It dispatches the
+request to the metamanager which saves this message in the local database.
+metamanager then sends an asynchronous message to edged. edged processes the insert request e,g.
+by starting the pod and populates the response in the message.
+metamanager inspects the message , extracts the response and sends it back to edged
+which sends it back to the cloud.
+
+## Update Operation
+`Update` operations can happen on objects  at the cloud / edge.
```
Comment (sids-b) [inline docs/modules/metamanager.md]: Please remove double space before at.
One more double space at line 33 before being

**Comment by m1093782566** [comment]
I would like to see the external link to this document for the sake of easily finding this document? Can we find a suitable place in README?

**Comment by rohitsardesai83** [comment]
> I would like to see the external link to this document for the sake of easily finding this document? Can we find a suitable place in README?



> I would like to see the external link to this document for the sake of easily finding this document? Can we find a suitable place in README?

The README already contains a link to the "modules" . metamanager , beehive and all other modules will be grouped under this folder. I think we can avoid putting too many links for each of the modules .

**Comment by m1093782566** [comment]
What about seting the link in https://github.com/kubeedge/kubeedge#introduction for each module?

**Comment by rohitsardesai83** [comment]
> What about setting the link in https://github.com/kubeedge/kubeedge#introduction for each component? I know beehive is an exception.

Yes that would be useful , but it would be better to update all the links in one pass , else the readme won't look good with links for some modules and no links for others.

**Comment by m1093782566** [comment]
> else the readme won't look good with links for some modules and no links for others.

I assume this work in ongoing?

**Comment by rohitsardesai83** [comment]
> > else the readme won't look good with links for some modules and no links for others.
> 
> I assume this work is ongoing?

Yes we should be able to finish this soon, maybe in a week.

**Pull request merged**